examples/plotting/file/grid.py

```diff
@@ -1,3 +1,10 @@
+'''  A grid plot that shows four figures that use different glyphs.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.layouts.gridplot, bokeh.plotting.Figure.line, bokeh.plotting.Figure.annular_wedge, bokeh.plotting.Figure.bezier, bokeh.plotting.Figure.quad
+    :refs: :ref:`userguide_plotting` > :ref:`userguide_plotting_bars_rects` > :ref:`userguide_layout_gridplot`
```

Comment:
We use the `>` to indicate a path through sections of the docs, but the layout section is not a subsection of `userguide_plotting_bars_rects` so this should be more like
```suggestion
    :refs: :ref:`userguide_plotting` > :ref:`userguide_plotting_bars_rects`, :ref:`userguide_layout` >  :ref:`userguide_layout_gridplot`
```

Reply:
cc @tcmetzger I don't think we actually provide any explicit guidance about how to author these, maybe the GFI itself would be a good place to elaborate